docsbox/docs/views.py

```diff
@@ -1,13 +1,24 @@
 import datetime
+import logging
 
 from flask import request, send_from_directory
-from flask_restful import Resource, abort
+from flask_restful import Resource, abort as flask_abort
 from requests import exceptions
 from docsbox import app
 from docsbox.docs.tasks import process_convertion, create_tmp_file_and_get_mimetype, get_task, do_task
 from docsbox.docs.utils import remove_extension, set_options, is_valid_uuid
 from docsbox.docs.via_controller import get_file_from_via  
 
+def abort(error_code, message, request):
+    if error_code >= 500:
+        error_level = logging.CRITICAL
+    elif error_code >= 400:
+        error_level = logging.ERROR
+    elif error_code >= 300:
```

Comment:
The 3xx HTTP status codes are for redirection. I think we don't need this here. If we want to log a warning we can do it outside the abort method

Reply:
fixed